appveyor.yml

```diff
@@ -10,6 +10,14 @@ environment:
 
   matrix:
     - TARGET_ARCH: x86
+      CONDA_PY: 27
+      CONDA_INSTALL_LOCN: C:\\Miniconda
+
+    - TARGET_ARCH: x64
+      CONDA_PY: 27
+      CONDA_INSTALL_LOCN: C:\\Miniconda-x64
+
+    - TARGET_ARCH: x86
```

Comment:
We don't really need to run the `unittests` for non compiled code. We trust that those are run upstream.
(It does not hurt to run them so I only skip the `win`+`py27` in order to get that package in the channel.)